pages/index.js

```diff
@@ -94,7 +94,7 @@ export default function Index(props) {
     } else if (actionToCarryOut === actions.fetchLatestWorkflowState) {
       fetchWorkflow();
     } else if (actionToCarryOut === actions.toggleCompleteStateLocally) {
-      updateWorkflowComplete(!workflow.currentTask.skipped, false);
+      updateWorkflowComplete(!workflow.currentTask.completed, false);
```

Comment:
Just checking... Should you remove the `!` here? Given completed kind of means the inverse of "skipped"

Reply:
Yep, see line `96`. We're looking to simply `toggleCompleteStateLocally` so we want to negate whichever value `workflow.currentTask.completed` is currently set to